FWCore/Common/interface/CudaDecls.h

```diff
@@ -0,0 +1,12 @@
+#ifndef FWCore_Common_interface_CudaDecls_h
```

Comment:
If I'm not wrong, these `#define`s are available from `#include <cuda_runtime.h>` .

If we would rather not include that, I would suggest to
  - move this file under `HeterogeneousCore/CUDAUtilities`
  - `#define` a single macro `CUDA_HOST_DEVICE` that expands to `__host__ __device`